config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_gkehubfeaturememberships.gkehub.cnrm.cloud.google.com.yaml

```diff
@@ -143,20 +153,23 @@ spec:
                             - external
                           required:
                           - name
+                          - kind
                         - not:
                             anyOf:
                             - required:
                               - name
                             - required:
                               - namespace
+                            - required:
+                              - kind
                           required:
                           - external
                         properties:
                           external:
-                            description: |-
-                              The Email of the Google Cloud Service Account (GSA) used for exporting Config Sync metrics to Cloud Monitoring. The GSA should have the Monitoring Metric Writer(roles/monitoring.metricWriter) IAM role. The Kubernetes ServiceAccount `default` in the namespace `config-management-monitoring` should be bound to the GSA.
-
-                              Allowed value: The `email` field of an `IAMServiceAccount` resource.
+                            description: The external name of the referenced resource
+                            type: string
+                          kind:
+                            description: Kind of the referent.
                             type: string
```

Comment:
Do you know why we are using the more detailed description here? It'd be nice if we can add it to a kubebuilder tag if that's needed. Ditto below for a few descriptions below.